presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/HiveCluster.java

```diff
@@ -26,6 +28,7 @@
 {
     /**
      * Create a connected {@link HiveMetastoreClient} to this HiveCluster
+     * @throws TException
      */
-    HiveMetastoreClient createMetastoreClient();
+    HiveMetastoreClient createMetastoreClient() throws TException;
```

Comment:
Code formatting nit: the `throws` should be in the next line, indented 8 spaces. See https://github.com/airlift/codestyle

Same goes for the rest of the PR.

(unfortunately, Checkstyle doesn't have a check we can use to enforce this)